<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADRES_NOT_DEFTERI_FISI</class>
 <widget class="QDialog" name="ADRES_NOT_DEFTERI_FISI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Not Defteri</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,60">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="navigation_buttons_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="20">
     <item row="0" column="0">
      <widget class="QLabel" name="label_sahis_firma_name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidget_not_defteri"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget_not_defteri</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
